test/wca_live/competitions_test.exs

```diff
@@ -28,6 +28,12 @@ defmodule WcaLive.CompetitionsTest do
     assert worlds.id in ids(list)
     assert open.id not in ids(list)
     assert euro.id in ids(list)
+
+    list = Competitions.list_competitions(%{filter: " championship  2022  "})
```

Comment:
```suggestion
    list = Competitions.list_competitions(%{filter: " championship  2020  "})
```

Reply:
I was about to commit the same fix, but github returned an error that you had already made the change 😅